File: .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexConvert.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace Microsoft.Recognizers.Text.DataTypes.TimexExpression
{
    public static class TimexConvert
    {
        public static string ConvertTimexToString(TimexProperty timex)
        {
            return TimexConvertEnglish.ConvertTimexToString(timex);
        }

        public static string ConvertTimexSetToString(TimexSet timexSet)
        {
            return TimexConvertEnglish.ConvertTimexSetToString(timexSet);
        }
    }
}

File: Python/libraries/datatypes-timex-expression/datatypes_timex_expression/timex_convert.py
#  Copyright (c) Microsoft Corporation. All rights reserved.
#  Licensed under the MIT License.

from .english import *


class TimexConvert:

    @staticmethod
    def convert_timex_to_string(timex):
        return convert_timex_to_string(timex)

    @staticmethod
    def convert_timex_set_to_string(timex):
        return convert_timex_set_to_string(timex)
